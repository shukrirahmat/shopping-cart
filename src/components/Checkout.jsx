import styles from "./Checkout.module.css"
import { Link } from "react-router-dom";

function Checkout() {
    return (

    <div className={styles.wrapper}>
        <p>Please enter your bank account number and PIN Code:</p>
        <div className={styles.inputContainer}>
            <label>Account number
            <input type="text" disabled/>
            </label>
            <label>PIN Code
            <input type="text" disabled/>
            </label>
        </div>
        <p>Just kidding! You should not do that.</p>
        <Link className={styles.return} to="/">Return to home</Link>
    </div>
    )
}

export default Checkout;