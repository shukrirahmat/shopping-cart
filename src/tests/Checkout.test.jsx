import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Checkout from "../components/Checkout";

describe("Checkout component", () => {
    it("renders checkout page", () => {
        const {container} = render(<BrowserRouter><Checkout/></BrowserRouter>);
        expect(container).toMatchSnapshot();
    })
})