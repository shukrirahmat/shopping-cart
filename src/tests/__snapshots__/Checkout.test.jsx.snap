// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Checkout component > renders checkout page 1`] = `
<div>
  <div
    class="_wrapper_c6267f"
  >
    <p>
      Please enter your bank account number and PIN Code:
    </p>
    <div
      class="_inputContainer_c6267f"
    >
      <label>
        Account number
        <input
          disabled=""
          type="text"
        />
      </label>
      <label>
        PIN Code
        <input
          disabled=""
          type="text"
        />
      </label>
    </div>
    <p>
      Just kidding! You should not do that.
    </p>
    <a
      class="_return_c6267f"
      href="/"
    >
      Return to home
    </a>
  </div>
</div>
`;
